### PR TITLE
Adding a ckETH page

### PR DESCRIPTION
**Pull request by domwoe**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/ethereum/cketh.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+## Overview
+
+Chain-key Ethereum (ckETH) is an
+[ICRC-2](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-2/README.md)-compliant
+token that is backed 1:1 by ETH.
+
+The ckETH functionality is provided through an interplay of a suite of canisters on ICP, and a
+helper contract on Ethereum:
+
+| Canister/Contract                                                                                   | Ethereum Mainnet                                                                                                                                                                    | Sepolia Testnet                                                                                                                                                                     |
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ethereum/cketh.mdx:14]:
```suggestion
| Canister / contract                                                                                   | Ethereum mainnet                                                                                                                                                                    | Sepolia testnet                                                                                                                                                                     |
```

File: docs/developer-docs/integrations/ethereum/cketh.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+It uses the following workflow:
+
+- Tokens are minted when a user calls the `deposit` function of the helper
+  contract on Ethereum with some ETH attached. The deposit function takes the
+  receiver's principal as an argument and emits a `ReceivedEth` event with the
+  `sender`, `value`, and `receiver` as payload.
+- The ckETH minter listens to the `ReceivedEth` event by periodically fetching
+  logs from multiple Ethereum JSON RPC cloud providers and mints the
+  corresponding amount of ckETH to the receiver's account on the ckETH ledger.
+- Users can withdraw by creating an ICRC-2 approval on the ledger and calling `withdraw_eth` with a receiver ETH address as an argument on the minter.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ethereum/cketh.mdx:32]:
```suggestion
- Users can withdraw ckETH by creating an ICRC-2 approval on the ledger and calling `withdraw_eth` with a receiver ETH address as an argument on the minter.
```

File: docs/developer-docs/integrations/ethereum/cketh.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+- The minter will then burn the ckETH on the ledger and create an Ethereum transaction and submit it to the Ethereum network.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ethereum/cketh.mdx:33]:
```suggestion
- The minter will then burn the ckETH on the ledger and create an Ethereum transaction that is submitted it to the Ethereum network.
```

File: docs/developer-docs/integrations/ethereum/cketh.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+The **ckETH ledger** and **ckETH index** canisters are regular ICRC-1 compliant canisters. A guide on how to interact with these canisters can be found in the [Interacting with an ICRC-1 ledger](https://internetcomputer.org/docs/current/developer-docs/integrations/icrc-1/interact-with-ICRC-1-ledger) section.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ethereum/cketh.mdx:35]:
```suggestion
The **ckETH ledger** and **ckETH index** canisters are regular ICRC-1 compliant canisters. A guide on how to interact with these canisters can be found in the [interacting with an ICRC-1 ledger](https://internetcomputer.org/docs/current/developer-docs/integrations/icrc-1/interact-with-ICRC-1-ledger) section.
```

File: docs/developer-docs/integrations/ethereum/cketh.mdx (hunk 2 above)
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ethereum/cketh.mdx:32]: Should we add a link to info on ICRC-2?

File: docs/developer-docs/integrations/ethereum/cketh.mdx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    src={useBaseUrl("/img/docs/cketh-withdraw.png")}
+    alt="Withdrawing ETH"
+    width="500"
+  />
+</div>
+
+## Integrating ckETH into a project
+
+For integrating ckETH into a dapp, you can follow the [generic instructions for ICRC-1 tokens](/docs/developer-docs/integrations/icrc-1/index.md).
+The `canisters` section in a `dfx.json` file of a project integrating with ckETH will look as follows:
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ethereum/cketh.mdx:65]:
```suggestion
To integrate with ckETH, the configuration of a project in the `dfx.json` file will include the following `canisters` section:
```

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/developer-docs/integrations/ethereum/cketh.mdx (hunk 3 above)
Comment (domwoe) [reply docs/developer-docs/integrations/ethereum/cketh.mdx:33]: Why do you want to have the 2nd part of the sentence passive when the first part is active?

**Comment by chmllr** [comment]
LGTM!

**Pull request merged**